fire-hose-management-frontend/src/components/Departments/DepartmentsDataGrid.jsx

```diff
@@ -28,67 +27,70 @@ export const DepartmentsDataGrid = ({
                                     }) => {
     const classes = useStyles();
 
-    const buttons = [
-        {
-            key: 1,
-            labelText: "Abteilung löschen",
-            icon: <DeleteIcon/>,
-            onClick: () => {
-                setDeleteOverlayVisibleFunction(true)
-            }
-        }, {
-            key: 1,
-            labelText: "Abteilung bearbeiten",
-            icon: <EditIcon/>,
-            onClick: () => {
-                setIsEditOverlayVisibleFunction(true)
-            }
-        }]
-
-
-    const buttonHandler = (props) => {
-        if (props) {
-            return (
-                <div>
-                    {
-                        buttons.map((btn) => {
-                            return (
-                                <IconButton
-                                    aria-label={btn.labelText}
-                                    onClick={(value) => {
-                                        btn.onClick(value)
-                                    }}>
-                                    {btn.icon}
-                                </IconButton>
-                            )
-
-                        })
-                    }
-                </div>
-            )
-        }
+    const getFullName = (params) => {
+        return `${params.row.contact.firstname} ${params.row.contact.lastname}`
     }
 
+    const getStreetAddress = (params) => {
+        return `${params.row.street} ${params.row.houseNumber}`
+    }
+
+    const getMail = (params) => {
+        return params.row.contact.mail
+    }
+
+
+    const editDepartment = React.useCallback(
+        (row) => () => {
+            selectedDepartmentFunction(row)
+            setIsEditOverlayVisibleFunction(true)
+        }, [selectedDepartmentFunction, setIsEditOverlayVisibleFunction]
+    )
+
+    const deleteDepartment = React.useCallback(
+        (row) => () => {
+            selectedDepartmentFunction(row)
+            setDeleteOverlayVisibleFunction(true)
+        }, [selectedDepartmentFunction, setDeleteOverlayVisibleFunction]
+    )
+
+
+    const columns = React.useMemo(
+        () => [
+            {field: "departmentName", headerName: "Abteilung", flex: true},
+            {field: "address", headerName: "Adresse", flex: true, valueGetter: getStreetAddress},
+            {field: "postalCode", headerName: "PLZ", flex: true},
+            {field: "location", headerName: "Ort", flex: true},
+            {field: "country", headerName: "Land", flex: true},
+            {field: "contact", headerName: "Ansprechpartner", flex: true, valueGetter: getFullName},
+            {field: "mail", headerName: "E-Mail", flex: true, valueGetter: getMail},
+            {
+                field: "actions",
+                type: "actions",
```

Comment:
Codacy has a fix for the issue: [Replace `················` with `········`](https://app.codacy.com/gh/StitZle/fire-hose-complete/pullRequest?prid=9542260)
```suggestion
        type: "actions",
```